Bump wildfly-jar-maven-plugin version to 9.0.0.Final

The default value for the `wildfly-jar-maven-plugin` version property is currently set to 8.0.1.Final but this does not allow to test against WildFly 28, errors massages showing up in the traces as following:

```
...
[ERROR] Failed to execute goal org.wildfly.plugins:wildfly-jar-maven-plugin:8.1.0.Final:package (bootable-jar-packaging) on project microprofile-config: Execution bootable-jar-packaging of goal org.wildfly.plugins:wildfly-jar-maven-plugin:8.1.0.Final:package failed: An API incompatibility was encountered while executing org.wildfly.plugins:wildfly-jar-maven-plugin:8.1.0.Final:package: java.lang.NoSuchMethodError: 'void org.jboss.galleon.universe.maven.MavenArtifact.addMetadata(java.lang.String, java.lang.String)'
[ERROR] -----------------------------------------------------
[ERROR] realm =    plugin>org.wildfly.plugins:wildfly-jar-maven-plugin:8.1.0.Final
[ERROR] strategy = org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy
[ERROR] urls[0] = file:/home/fburzigo/Temp/wildfly-28-test/local-maven-repository/org/wildfly/plugins/wildfly-jar-maven-plugin/8.1.0.Final/wildfly-jar-maven-plugin-8.1.0.Final.jar
[ERROR] urls[1] = file:/home/fburzigo/Temp/wildfly-28-test/local-maven-repository/org/twdata/maven/mojo-executor/2.3.1/mojo-executor-2.3.1.jar
...
```

* Steps to reproduce:

`mvn clean verify -Dts.bootable -Djboss.dist=/home/fburzigo/Temp/wildfly-28-test/server -Dtest=RegisterConfigAndReleaseConfigMethodsTest`

--- a/pom.xml
+++ b/pom.xml
@@ -79,7 +79,7 @@
         <testsuite.galleon.pack.artifactId>wildfly-galleon-pack</testsuite.galleon.pack.artifactId>
         <testsuite.galleon.pack.version>24.0.0.Final</testsuite.galleon.pack.version>
         <!-- Bootable JAR -->
-        <version.org.wildfly.jar.plugin>8.1.0.Final</version.org.wildfly.jar.plugin>
+        <version.org.wildfly.jar.plugin>9.0.0.Final</version.org.wildfly.jar.plugin>
         <version.org.bitbucket.b_c.jose4j>0.7.4</version.org.bitbucket.b_c.jose4j>
         <!-- Modular JDK JPMS options -->
         <server.jvm.jpms.args>--add-exports=java.desktop/sun.awt=ALL-UNNAMED --add-exports=java.naming/com.sun.jndi.ldap=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.lang.invoke=ALL-UNNAMED --add-opens=java.base/java.lang.reflect=ALL-UNNAMED --add-opens=java.base/java.io=ALL-UNNAMED --add-opens=java.base/java.security=ALL-UNNAMED --add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.base/java.util.concurrent=ALL-UNNAMED --add-opens=java.management/javax.management=ALL-UNNAMED --add-opens=java.naming/javax.naming=ALL-UNNAMED</server.jvm.jpms.args>

